src/storage/transaction/ChainAddEdgeProcessor.h

```diff
@@ -0,0 +1,88 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#pragma once
+
+#include "storage/transaction/BaseChainProcessor.h"
+
+namespace nebula {
+namespace storage {
+
+class ChainAddEdgesProcessor : public BaseChainProcessor {
+    using LockGuard = nebula::MemoryLockGuard<std::string>;
```

Comment:
`using LockGuard = nebula::MemoryLockGuard<std::string>;`  can move to BaseChainProcessor ?

Reply:
good catch.